tools/compute-model-sizes.sh

```diff
@@ -0,0 +1,7 @@
+echo 'FROM registry.access.redhat.com/ubi9-minimal:9.3' >Containerfile
+echo 'RUN microdnf install -y git make g++' >>Containerfile
+echo 'RUN git clone https://github.com/ggerganov/llama.cpp' >>Containerfile
+echo 'RUN cd llama.cpp && make simple' >>Containerfile
+echo 'RUN pwd' >>Containerfile
```

Comment:
Drive-by comment:
There's a more idiomatic way of creating a file than using `echo` via

```bash
cat >Containerfile<< EOF
# Contents of the file
EOF
```

This pattern is used a lot in Podman's system tests, for instance here:
https://github.com/containers/podman/blob/2aad385b51481008e337ec17950cfaace553ad79/test/system/255-auto-update.bats#L489-L496